src/configuration/app-containers.md

```diff
@@ -123,3 +123,27 @@ web:
 ## Upgrading from previous versions of the configuration file.
```

Comment:
I don't think this is the right page for this information.  It should be with the other UI-tour pages.  The /configuration/ section is all about editing YAML files.